File: 1938/1938.cs
public class Solution {
    public int MaximumGeneticDifferenceQuery(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(MaximumGeneticDifferenceQuery(root.left), MaximumGeneticDifferenceQuery(root.right));
    }
}

File: 1938/1938.go
func maximumGeneticDifferenceQuery(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := maximumGeneticDifferenceQuery(root.Left)
	r := maximumGeneticDifferenceQuery(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}